### ISSUE TITLE
Can't finish generating.

### ISSUE THREAD

**Issue by saxobroko** [luka1199/geo-heatmap#7]
Getting this in the console.
```
Loading data from Location History.json...
|##################################################################################################|100% Time:  0:00:01
Traceback (most recent call last):
  File "geo_heatmap.py", line 85, in <module>
    generator.run(data_file, output_file)
  File "geo_heatmap.py", line 73, in run
    self.loadData(data_file)
  File "geo_heatmap.py", line 30, in loadData
    lat = round(loc["latitudeE7"] / 1e7, 6)
KeyError: 'latitudeE7'
```

**Comment by luka1199** [comment]
Can you check if you have any data in your location history json file? That could be a reason for this error

**Comment by saxobroko** [comment]
There is location data

**Comment by luka1199** [comment]
Can you send me the basic format of the file?

**Comment by saxobroko** [comment]
{
  "locations" : [ {
    "timestampMs" : "#############",
    "latitudeE7" : -##,
    "longitudeE7" : ###,
    "accuracy" : ##,
    "activity" : [ {
      "timestampMs" : "##,
      "activity" : [ {
        "type" : "STILL",
        "confidence" : ##
      }, {
        "type" : "UNKNOWN",
        "confidence" : ##
      }, {
        "type" : "IN_VEHICLE",
        "confidence" : ##
      }, {
        "type" : "ON_FOOT",
        "confidence" : ##
      }, {
        "type" : "UNKNOWN",
        "confidence" : ##
      } ]
    }, {
      "timestampMs" : "###",
      "activity" : [ {
        "type" : "UNKNOWN",
        "confidence" : ##
      }, {
        "type" : "STILL",
        "confidence" : ##
      }, {
        "type" : "IN_VEHICLE",
        "confidence" : ##
      } ]
    }
}

Thats it very basic. the real file has 3 million lines.

**Comment by luka1199** [comment]
For some reason the latitude and longitude in your second location is missing.

**Comment by luka1199** [comment]
I think I fixed it with ca8deaa. Can you `git pull` and check if it works now?
